packages/cli/src/lib/cmds/shared.ts

```diff
@@ -84,3 +88,58 @@ export async function updateVersions(
     appVersionName: appVersionName,
   };
 }
+
+export async function generateManifestChecksumFile(manifestFile: string,
+    prompt: Prompt): Promise<void> {
+  fs.readFile(manifestFile, async function(err, data) {
```

Comment:
Could you use the promise version of `readFile` here please?

Reply:
Actually, I think the current version has a bug - `await generateManifestChecksumFile(...)` won't actually wait for the completion of the callback in this promise.

Reply:
Yep, you're absolutely right. Changed it to use the promise version.